sdk-api-src/content/handleapi/nf-handleapi-duplicatehandle.md

```diff
@@ -76,30 +76,32 @@ The handle to be duplicated. This is an open object handle that is valid in the
 
 A handle to the process that is to receive the duplicated handle. The handle must have the PROCESS_DUP_HANDLE access right.
 
+This parameter is optional and can be specified as NULL if the **DUPLICATE_CLOSE_SOURCE** flag is set in _Options_.
+
 ### -param lpTargetHandle [out]
 
 A pointer to a variable that receives the duplicate handle. This handle value is valid in the context of the target process. 
 
-
-
-
-If <i>hSourceHandle</i> is a pseudo handle returned by <a href="/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess">GetCurrentProcess</a> or <a href="/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getcurrentthread">GetCurrentThread</a>, <b>DuplicateHandle</b> converts it to a real  handle to a process or thread, respectively.
+If <i>hSourceHandle</i> is a pseudo handle returned by <a href="/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess">GetCurrentProcess</a> or <a href="/windows/desktop/api/processthreadsapi/nf-processthreadsapi-getcurrentthread">GetCurrentThread</a>, <b>DuplicateHandle</b> converts it to a real handle to a process or thread, respectively.
 
 If <i>lpTargetHandle</i> is <b>NULL</b>, the function duplicates the handle, but does not return the duplicate handle value to the caller. This behavior exists only for backward compatibility with previous versions of this function. You should not use this feature, as you will lose system resources until the target process terminates.
 
+This parameter is ignored if _hTargetProcessHandle_ is **NULL**.
```

Comment:
It makes sense that the parameters related to the actual duplication: `lpTargetHandle`, `dwDesiredAccess`, and `bInheritHandle`, are not used when `hTargetProcessHandle` is **NULL**. I can't (as easily) confirm this without having the source code.